couchbase-ansible/azuredeploy.json

```diff
@@ -449,25 +287,25 @@
 		{
 			"apiVersion": "2015-05-01-preview",
 			"type": "Microsoft.Network/virtualNetworks",
-			"name": "[variables('newVirtualNetworkName')]",
+			"name": "[variables('virtualNetworkName')]",
 			"location": "[parameters('location')]",
 			"properties": {
 				"addressSpace": {
 					"addressPrefixes": [
-						"[parameters('addressPrefix')]"
+						"[variables('addressPrefix')]"
 					]
 				},
 				"subnets": [
 					{
-						"name": "[parameters('subnet1Name')]",
+						"name": "[variables('subnet1Name')]",
 						"properties": {
-							"addressPrefix": "[parameters('subnet1Prefix')]"
+							"addressPrefix": "[variables('subnet1Prefix')]"
 						}
 					},
 					{
-						"name": "[parameters('subnet2Name')]",
+						"name": "[variables('subnet2Name')]",
```

Comment:
I wanted to show how to create multiple subnets and I also wanted to force users to think about where to place the VMS. Otherwise we might be biasing the users to only use one Subnet?